config/software/supermarket.rb

```diff
@@ -15,10 +15,11 @@
 #
 
 name "supermarket"
-default_version "master"
+default_version 'master'
```

Comment:
Okay :) I changed it back to double quotes so it doesn't look like a gratuitous diff